WalletWasabi.Fluent/ViewModels/ApplicationViewModel.cs

```diff
@@ -51,6 +49,20 @@ public ApplicationViewModel(IMainWindowService mainWindowService)
 
 	public ICommand QuitCommand { get; }
 
+	private void AboutExecute()
+	{
+		MainViewModel.Instance.DialogScreen.To(
+			new AboutViewModel(navigateBack: MainViewModel.Instance.DialogScreen.CurrentPage is not null));
+	}
+
+	private IObservable<bool> AboutCanExecute()
+	{
+		return MainViewModel.Instance.DialogScreen
+			.WhenAnyValue(x => x.CurrentPage)
+			.ObserveOn(RxApp.MainThreadScheduler)
+			.Select(x => x is null);
+	}
```

Comment:
No, this is not about terminating.

We only allow opening the About page when the `DialogScreen.CurrentPage` is null, meaning there is no dialog opened. The reason for this is that it can override what the user is doing.

BTW the parameter of AboutViewModel:
```c#
navigateBack: MainViewModel.Instance.DialogScreen.CurrentPage is not null
```
is a leftover. It seems to me it makes no sense as it will be always false due to canExecute logic.

The only thing that @wieslawsoltes changed (apart from refactoring to make the code more readable) is the `.ObserveOn(RxApp.MainThreadScheduler)` line, so it will always happen on the UI thread.